### PR TITLE
Translate Performance chapter to Spanish

### PR DESCRIPTION
**Pull request by JMPerez**
This is my take to translate the Performance chapter to Spanish language.

### CODE REVIEW

File: src/content/es/2019/performance.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+Los resultados de ECT anteriores parecen sugerir que existe una correlación entre la velocidad de conexión y el rendimiento del FID. A medida que disminuye la velocidad de conexión efectiva de los usuarios, el porcentaje de sitios web en los que experimentan FID rápido también disminuye: el 41% de los sitios web visitados por los usuarios con un ECT 4G tienen FID rápido, el 22% con 3G, el 19% con 2G y el 15% con 2G lento.
+
+#### FID por geo
+
+<figure id="fig18">
+<iframe width="600" height="940" seamless frameborder="0" scrolling="no" src="https://docs.google.com/spreadsheets/d/e/2PACX-1vSQlf3_ySLPB5322aTumUZhbVGdaUdkmi1Hs4bYuO3Z1kqM4xspx7REbwXukwPd_tsOSg6oImzpYLM9/pubchart?oid=11500240&amp;format=interactive"></iframe>
+<figcaption>Figura 18. Distribución de sitios web etiquetados como que tienen un FID rápido, moderado o lento, desglosados por geo.</figcaption>
+</figure>
+
+En este desglose del FID por ubicación geográfica, Corea está nuevamente al frentes. Pero los principales geos tienen algunas caras nuevas: Australia, Estados Unidos y Canadá son los siguientes, con más del 50% de los sitios web que tienen una FID rápida.
```
Comment (c-torres) [inline src/content/es/2019/performance.md]:
```suggestion
En este desglose del FID por ubicación geográfica, Corea está nuevamente al frente. Pero los principales geos tienen algunas caras nuevas: Australia, Estados Unidos y Canadá son los siguientes, con más del 50% de los sitios web que tienen una FID rápida.
```

File: src/content/es/2019/performance.md [hunk 2]
```diff
@@ -0,0 +1,216 @@
+---
+part_number: II
+chapter_number: 7
+title: Rendimiento
```
Comment (c-torres) [inline src/content/es/2019/performance.md]:
```suggestion
title: Performance
/```

Comment (rviscomi) [reply src/content/es/2019/performance.md]: This is probably a bug, but I agree that we should be consistent for now and we can fix this later.

**Comment by rviscomi** [comment]
@JMPerez  @c-torres make sure you're both included in the [2019.json](https://github.com/HTTPArchive/almanac.httparchive.org/blob/master/src/config/2019.json) list of contributors and your membership includes the `translators` team. Thank you for your help with this!

**Pull request merged**